frontend/src/app/modules/common/hide-section/show-section-dropdown.component.ts

```diff
@@ -30,8 +30,10 @@ import {HideSectionService} from "./hide-section.service";
 import {Component, ElementRef, OnInit} from "@angular/core";
 import {DynamicBootstrapper} from "core-app/globals/dynamic-bootstrapper";
 
+export const showSectionDropdownSelector = 'show-section-dropdown';
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/3b1eae5a64159f8a044579a25870c772186dbd3f/frontend/src/app/modules/common/hide-section/show-section-dropdown.component.ts#L34"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>